[2024-05-13T03:55:05.308+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T03:55:05.325+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-03-21T00:00:00+00:00 [queued]>
[2024-05-13T03:55:05.331+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-03-21T00:00:00+00:00 [queued]>
[2024-05-13T03:55:05.332+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-13T03:55:05.345+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): greet> on 2022-03-21 00:00:00+00:00
[2024-05-13T03:55:05.348+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3574) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-13T03:55:05.350+0000] {standard_task_runner.py:63} INFO - Started process 3578 to run task
[2024-05-13T03:55:05.349+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'greet', 'scheduled__2022-03-21T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp8w_a5g0w']
[2024-05-13T03:55:05.351+0000] {standard_task_runner.py:91} INFO - Job 202: Subtask greet
[2024-05-13T03:55:05.376+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-03-21T00:00:00+00:00 [running]> on host f1ddc6e75993
[2024-05-13T03:55:05.415+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mani' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2022-03-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-21T00:00:00+00:00'
[2024-05-13T03:55:05.416+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T03:55:05.424+0000] {logging_mixin.py:188} INFO - Hello World
[2024-05-13T03:55:05.424+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-13T03:55:05.425+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T03:55:05.430+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=greet, run_id=scheduled__2022-03-21T00:00:00+00:00, execution_date=20220321T000000, start_date=20240513T035505, end_date=20240513T035505
[2024-05-13T03:55:05.448+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T03:55:05.457+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-13T03:55:05.458+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
